exodus/analysis_query/views.py

```diff
@@ -52,7 +77,16 @@ def json(request, r_id):
         raise Http404(_("AnalysisRequest does not exist"))
     r.bucket = ''
     r.description = _(r.description)
```

Comment:
I think these 2 lines are not necessary anymore

Reply:
I don't understand why :/

Reply:
Now that you return a `JsonResponse` object based on the dictionary `obj` and that you create `obj` directly with the correct description and bucket, we don't need to modify the `r` object